e2e-tests/upgrade-sharded/compare/statefulset_upgrade-sharded-rs0-oc.yml

```diff
@@ -1,204 +0,0 @@
-apiVersion: apps/v1
```

Comment:
Are you removing these `upgrade-sharded` files because they are not used?

Reply:
I don't see where these two files used

Reply:
me too, but let's check the logic: https://jira.percona.com/browse/K8SPSMDB-358
so please just leave them in place until this ticket is evaluated

Reply:
it's a good find ;)

Reply:
fixed, please check again